deploy-storybook/action.yml

```
@@ -137,10 +162,12 @@ runs:
       uses: actions/github-script@v3.0.0
       env:
         RELEASE: ${{ steps.repo.outputs.release_name }}
+        PR_NUMBER: ${{ steps.repo.outputs.pr_number }}
```

```suggestion
        PR_NUMBER: ${{ inputs.pr-number }}
```

---

I think we lost `github.event.number || github.event.issue.number`

---

Its done on line 172, but its better to have it in one place I think:
```suggestion
        PR_NUMBER: ${{ github.event.number || github.event.issue.number || inputs.pr-number }}
```

---

`issue number` is destructed from `context` so i did not want to decide it on 165